src/app/forms/components/date/date.component.html

```diff
@@ -10,9 +10,11 @@
           <input
             #dayEl="ngModel"
             appNumberOnly
+            appFocusNext
+            [includeTime]="includeTime"
             class="govuk-input govuk-date-input__input govuk-input--width-2"
             [ngClass]="{ 'govuk-input--error': elementHasErrors(0) && (dayEl.dirty || dayEl.touched) }"
-            id="{{ name }}-day"
+            attr.id="{{ name }}-day"
```

Comment:
Why the change? especially since the other components don't have that change?
```suggestion
            id="{{ name }}-day"
```